### DISCUSSION TITLE
conda-forge package

### DISCUSSION THREAD

**Discussion by davidbrochart** [frankie567/httpx-ws#18] [Q&A]
PR is [here](https://github.com/conda-forge/staged-recipes/pull/21636) (it also needs a [feedstock for hatch-regex-commit](https://github.com/conda-forge/staged-recipes/pull/21637)).
Although [httpx requires Python >=3.7](https://github.com/encode/httpx/blob/f11eff45b7d53a6bbd17d453380b1e0ebf494102/pyproject.toml#L9), [httpx-feedstock requires Python >=3.8](https://github.com/conda-forge/httpx-feedstock/blob/40bd1344cdfe2e7e15b39197f6ccf5ffaf005e18/recipe/meta.yaml#L26), so `httpx-ws-feedstock` will do the same, which means that it won't require [typing-extensions](https://github.com/frankie567/httpx-ws/blob/e5afdb1a5108c9d659b23a3fbe72dceac883f97c/pyproject.toml#L81).

**Comment by frankie567** [comment]
Thank you for this, David 🙏 Does it need an action from my side?

**Reply by davidbrochart** [reply]
No, I just wanted to let you know. Do you want to be added as a maintainer of these packages?

**Reply by frankie567** [reply]
Yes, that could be useful if someday we need to make some changes there :)

**Reply by davidbrochart** [reply]
OK, I will ping you on the PRs, since you need to agree there.
